src/app/admin/admin-products/admin-products.component.html

```diff
@@ -1 +1,21 @@
-<a routerLink="/admin/products/new" class="btn btn-primary">New Product</a>
+<p>
+  <a routerLink="/admin/products/new" class="btn btn-primary">New Product</a>
+</p>
+<table class="table">
+  <thead>
+    <tr>
+      <th>Title</th>
+      <th>Price</th>
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr *ngFor="let product of products$ | async">
```

Comment:
This will be changed to the productsObservable or whatever you name it

Reply:
Done